index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body class="container">
+    <div class="header_container">
+        <div>
+            <input type="image" src="MyLogo.svg" alt="MyLogo">
+        </div>
+        <img src="menu.svg" alt="Menu">
+    </div>
+    <img class="arc" src="arc.svg" width="1" height="15" alt="arc">
+    <img class="bar" src="bar.svg" width="1" height="15" alt="bar">
+    <img class="c" src="c.svg" width="1" height="15" alt="curve">
+    <img class="circle" src="circle.svg" width="1" height="15" alt="circle">
+    <div>
+        <h2 class="info_box">
+            I'm Zedd Glad to see you!
+        </h2>
+    </div>
+    <div>
+        <p class="body_section">
+        I'am a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+        </p>
+    </div>
+    <div>
+        <p class="contact_section"><a href="#">LET'S CONNECT</a></p>
+    </div>
+    <div>
+    <img src="cat.svg" width="1" height="15" alt="cat">
+    <img src="bird.svg" width="1" height="15" alt="bird">
+    <img src="m.svg" width="1" height="15" alt="M">
+    <img src="n.png" width="1" height="15" alt="N">
+    <img src="two.svg" width="1" height="15" alt="two">
+    </div>
```

Comment:
- [ ] [REQUIRED] Please, make sure t use a `ul` with `li` children instead of `div` and `img`.

example:

```html
<ul>
  <li><a href="#"><img src="m.svg"/></a></li>
</ul>
```